### ISSUE TITLE
[A11Y][LCW][REACT] There is multiple 'region landmarks'.

### ISSUE THREAD

**Issue by elopezanaya** [microsoft/BotFramework-WebChat#4733]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

for LCW V2 (React version), has been raised an A11Y bug for no necessary landmarks in the UI, including the region landmark in the webchat_toaster component.

https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3339415

this is a copy of the bug for LCW v1.0 https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3108707/

We have fixed the issue for the elements in LCW V2.0, but the missing one exists on webchat

<img width="958" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/981914/5d9578eb-4962-4251-9dc5-89993a2430d6">

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS, macOS: Safari with VoiceOver

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

I am not using latest

### Which area does this issue affect?

Live region

### What is the public URL for the website?

 https://lcwtestblob.blob.core.windows.net/testsrc/landmarks/dist/test.html

### How to reproduce the issue?

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
Environment Details:
Application Name: Microsoft Dynamics 365 Omnichannel Online Chat – CRA Online Chat Agent
#URL: Omnichannel Agent Dashboard
Windows Version: Win10
Chromium edge: Version 110.0.1556.0 (Official build) dev (64-bit)
Repro Steps:
Launch the application using URL https://dailymonlychattp20512.crm10.dynamics.com/main.aspx and login with appropriate credentials.
"Dynamics 365" app should open. TAB to "Customer Service admin center" and hit ENTER key.
TAB to "Workstreams" tab present under the "Customer Support" and hit ENTER key. TAB to any chat and hit ENTER key. Copy the chat snippet present under "Chat widget" section.
Open the URL: https://www.w3schools.com/code/tryit.asp?filename=GOE65U628QHW and paste the code snippet and run the code. 
Once the code is run, you will be able to see chat widget. TAB to chat widget and hit ENTER key. TAB to required edit field control and edit something, TAB to "Submit" button and hit ENTER key, chat section will appear.
Now go back to "Dynamics 365" application, TAB to "Omnichannel for Customer Service" app present under "Published apps" and hit ENTER key.  "Omnichannel for Customer Service" screen should open.
You will be able to incoming chat notification and then TAB to "Accept" button and hit ENTER key. Once the Visitor has been accepted, you will be able to see chat section.
TAB to "Type a public message" edit and edit some message, hit ENTER key, once the message sent.
Once the conversation end, go to W3C chat window. Open Accessibility insights for Web and enable Landmark Toggle.
Observe that landmarks are not properly defined on the 'Chat' window.
Actual:
Landmark are not properly defined in the 'Chat' window.
There are no banner and main landmarks
There is multiple 'region landmarks'.
Unnecessary complementary landmark is defined.
Expected:
Landmarks should be properly defined in the 'Chat' window.
Banner and main landmarks should be defined. Banner for the logo region. and main landmark for the main content of the page.
There should not be multiple 'region landmarks'.
User Impact:
If too many regions and landmarks are defined, the user will not be able to navigate sequentially and quickly. It doesn’t solve the purpose of quick navigation and Users efficiency will be reduced.
MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)

NOTE: "Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"

### What do you expect?

Expected:
Landmarks should be properly defined in the 'Chat' window.
Banner and main landmarks should be defined. Banner for the logo region. and main landmark for the main content of the page.
There should not be multiple 'region landmarks'.
User Impact:
If too many regions and landmarks are defined, the user will not be able to navigate sequentially and quickly. It doesn’t solve the purpose of quick navigation and Users efficiency will be reduced.

### What actually happened?

Actual:
Landmark are not properly defined in the 'Chat' window.
There are no banner and main landmarks
There is multiple 'region landmarks'.
Unnecessary complementary landmark is defined.

### Do you have any screenshots or recordings to repro the issue?

<img width="958" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/981914/5d9578eb-4962-4251-9dc5-89993a2430d6">

### Did you find any DOM elements that might have caused the issue?

highlighed in the screenshot

### MAS reference

https://microsoft.sharepoint.com/:w:/s/accessibility/EXdDUZzldv5MgwDaMHnTzcMBcT8opKGncYjU-gCJ4uqQdg?e=9nuVmt
